### DISCUSSION TITLE
Ajaxpro add to cart popup is not showing on live

### DISCUSSION THREAD

**Discussion by nilaykumardeveloper** [breezefront/community#238] [1. Help]
@vovayatsyuk 
we are using a breeze theme and when we upgraded magento2.4.5-p5 from magento2.4.5-p2. after that add to cart success poup is not showing on PDP page.
PDP URL: https://www.creative-activity.com/the-essential-resilience-and-wellbeing-toolkit-for-early-years-and-younger-children.html.
We also upgraded all swissup modules locally but still same issue.
Kindly look on this and suggest if we can do anything.

All Swissup modules version:
        "swissup/address-autocomplete": "^1.6",
        "swissup/ajaxlayerednavigation": "^1.3",
        "swissup/ajaxpro": "^1.6",
        "swissup/breeze-blank": "^2.0",
        "swissup/module-breeze-mirasvit-gtm": "^1.0",
        "swissup/module-countdowntimer": "^1.0",
        "swissup/module-marketplace": "^1.9",
        "swissup/navigationpro": "^1.16",
        "swissup/pagespeed": "^1.6",
        "swissup/pro-labels": "^1.6",

**Comment by vovayatsyuk** [comment]
Looks like this commit should fix the issue: https://github.com/breezefront/module-breeze/commit/83b195bc0f0b999035f481789abccd3a30dbd2a1

**Comment by nilaykumardeveloper** [comment]
@vovayatsyuk yes, it is fixed. Thank you :)
